libmscore/property.h

```diff
@@ -284,6 +284,7 @@ enum class Pid {
       FONT_FACE,
       FONT_SIZE,
       FONT_STYLE,
+      TEXT_LINE_SPACING,
```

Comment:
Do I have to update this count? (see line 294)

Reply:
better

Reply:
Done. The count was outdated and also unsynched with .cpp